shuup/gdpr/receivers.py

```diff
@@ -31,4 +33,8 @@ def create_consents_login_allowed(sender, request, user, *args, **kwargs):
 
 @receiver(checkout_complete)
 def create_consents_checkout_complete(sender, request, user, order, *args, **kwargs):
-    create_user_consent_for_all_documents(request.shop, user)
+    gdpr_settings = GDPRSettings.get_for_shop(request.shop)
+    if not user.is_authenticated() and has_installed("shuup.gdpr") and gdpr_settings.enabled:
```

Comment:
Well under gdpr app we do not need to check whether it is installed. Likely the enabled check should also be under `create_user_consent_for_all_documents` not sure this check needs to be added either. Maybe instead of this line should just be `reate_user_consent_for_all_documents(request.shop, order.orderer.user)`.